sdks/python/apache_beam/transforms/environments.py

```diff
@@ -66,7 +67,8 @@
     'EmbeddedPythonEnvironment',
     'EmbeddedPythonGrpcEnvironment',
     'SubprocessSDKEnvironment',
-    'RunnerAPIEnvironmentHolder'
+    'RunnerAPIEnvironmentHolder',
```

Comment:
I don't think 'RunnerAPIEnvironmentHolder' is used anymore. Can you just delete that reference and the definition below ?

Reply:
done.